src/psyclone/dynamo0p3.py

```diff
@@ -6193,31 +7273,17 @@ def field_bcs_kernel(self, function_space):
                 "Expected a gh_field from which to look-up boundary dofs "
                 "for kernel {0} but got {1}".format(self._kern.name,
                                                     farg.type))
-        new_parent, position = parent.start_parent_loop()
-        new_parent.add(AssignGen(new_parent, pointer=True,
-                                 lhs="boundary_dofs",
-                                 rhs=farg.proxy_name +
-                                 "%vspace%get_boundary_dofs()"),
-                       position=["before", position])
+        self._arglist.append("boundary_dofs_"+farg.name)
```

Comment:
is there a namespace manager requirement here?

Reply:
and again.